src/js/injectBaseStylesheet.js

```diff
@@ -63,5 +63,6 @@ export default function injectBaseStylesheet() {
   const head = document.head;
 
   const allHeadElements = head.getElementsByTagName("*");
-  allHeadElements.innerHTML = styleEl + allHeadElements.innerHTML;
+  const styleTag = allHeadElements.item(allHeadElements.length - 1);
+  styleTag.innerHTML = styleEl.innerHTML + styleTag.innerHTML;
```

Comment:
This will definitely work in the context of the local demo since the last child element happens to be a `<style>` element, but I don't think we can assume that will be the case for all of our users. We should prepend a completely separate `<style>` element into the list of `allHeadElements` to ensure we don't accidentally cause any errors.

Reply:
```suggestion
  const styleTag = allHeadElements.item(allHeadElements.length - 1);
  styleTag.outerHTML = styleEl.outerHTML + styleTag.outerHTML;
```

I don't know if this is the correct way to do this... but it works 🤷